client/asset/eth/multirpc.go

```diff
@@ -1242,10 +1244,11 @@ const (
 	providerInfura      = "infura.io"
 	providerRivetCloud  = "rivet.cloud"
 	providerAlchemy     = "alchemy.com"
+	providerAnkr        = "ankr.com"
+	providerBlast       = "blastapi.io"
 
 	// Non-compliant providers
 	providerCloudflareETH = "cloudflare-eth.com" // "SuggestGasTipCap" error: Method not found
-	providerAnkr          = "ankr.com"           // "SyncProgress" error: the method eth_syncing does not exist/is not available
```

Comment:
I keep meaning to come back to this, but even before this diff, I had been using ankr and it was never treated as non-compliant nor did it have to run a compliance check (I searched the logs, and I don't have a compliant-providers.json anywhere on my drive).  Not sure how it was workin.  The host was https://rpc.ankr.com/eth_goerli